timeseries/src/autogluon/timeseries/models/abstract/abstract_timeseries_model.py

```diff
@@ -191,27 +320,46 @@ def _get_default_auxiliary_params(self) -> dict:
             max_time_limit=None,
         )
 
-    def initialize(self, **kwargs) -> dict:
-        # TODO: remove **kwargs from method signature
-        # TODO: do we even need deferred initialization?
+    # TODO: remove
+    @classmethod
+    def _get_default_ag_args(cls) -> dict:
+        return {}
+
+    def _init_params(self):
+        """Initializes model hyperparameters"""
+        hyperparameters = self._user_params
+        self.nondefault_params = []
+        if hyperparameters is not None:
+            self.params.update(hyperparameters)
+            self.nondefault_params = list(hyperparameters.keys())[
+                :
+            ]  # These are hyperparameters that user has specified.
```

Comment:
Nanonit: Can we move the comment above? The formatting hurts my eyes a bit.
```suggestion
            # These are hyperparameters that user has specified.
            self.nondefault_params = list(hyperparameters.keys())[:]  
```